### DISCUSSION TITLE
I want my video call to float on top of other objects

### DISCUSSION THREAD

**Discussion by ILMidnight** [spatialsys/spatial-unity-sdk#28] [Help] [answered]
Currently, when using the cam, the cam screen is displayed in a circle above the character's nickname, and I want to create boxes to change the texture. How can I do this?

**Comment by AlanSpatial** [comment, marked as answer]
Unfortunately its not possible to change the position of the webcams inside spatial right now.

**Reply by ILMidnight** [reply]
That's too bad.
The cam view above the character's head seems to be implemented inside the canvas, so is it impossible to import that data into Unity?
